internal/controller/csiaddons/csiaddonsnode_controller.go

```diff
@@ -105,7 +105,9 @@ func (r *CSIAddonsNodeReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 	// namespace + "/" + leader identity(pod name) is the key for the connection.
 	// this key is used by GetLeaderByDriver to get the connection
-	key := csiAddonsNode.Namespace + "/" + podName
+	// csi-lib-utils/leaderelection:sanitizeName() is used to sanitize the leader identity used for the leases
```

Comment:
`csi-lib-utils/leaderelection:sanitizeName()` is never called, that makes the comment very confusing. Maybe mention `util.NormalizeLeaseName()` instead?

Reply:
its called internall when we create the leader election object using csi-lib-utils packages, i updated to mention to use util.NormalizeLeaseName(). PTAL